apps/web/src/hooks/useVoiceMode.ts

```
@@ -146,26 +193,46 @@ export function useVoiceMode({
+  }, []);
+
+  // Stop barge-in speech detection that listens while TTS is playing.
+  const stopBargeInMonitoring = useCallback(() => {
+    if (bargeInRefs.current.animationFrame) {
+      cancelAnimationFrame(bargeInRefs.current.animationFrame);
+      bargeInRefs.current.animationFrame = null;
     }
-    return audioContextRef.current;
+
+    if (bargeInRefs.current.stream) {
+      bargeInRefs.current.stream.getTracks().forEach((track) => track.stop());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)